src/client/datascience/context/interpreterCountTracker.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { IExtensionSingleActivationService } from '../../activation/types';
+import { inject, injectable } from 'inversify';
+import { IInterpreterService } from '../../interpreter/contracts';
+import { IPythonExtensionChecker } from '../../api/types';
+import { noop } from '../../common/utils/misc';
+
+@injectable()
+export class InterpreterCountTracker implements IExtensionSingleActivationService {
+    private static interpreterCount = 0;
+    public static get totalNumberOfInterpreters() {
+        return InterpreterCountTracker.interpreterCount;
+    }
+    constructor(
+        @inject(IPythonExtensionChecker) private readonly pythonExtensionChecker: IPythonExtensionChecker,
+        @inject(IInterpreterService) private readonly interpreterService: IInterpreterService
+    ) {}
+    public async activate() {
+        if (!this.pythonExtensionChecker.isPythonExtensionInstalled) {
```

Comment:
Should this refresh when the python extension does get installed?

Reply:
Fixed in another place, good pick.
Done